setup.py

```diff
@@ -127,15 +127,15 @@ def run(self):
     'jsmin',
   ],
   install_requires=[
-    'Django < 1.7',
+    'Django ~= 1.11',
     'humanize',
     'parsimonious == 0.5',
     'parsedatetime ~= 1.0',
   ],
   tests_require=[
     'tox',
     'nose',
-    'django-nose < 1.4',
+    'django-nose ~= 1.4',
```

Comment:
we con't have to tie this to 1.4; we can go higher, since we're at dj 11 now, we don't need an upper limit

Reply:
:+1: